Modules/IndividualAssessmentFormPool/classes/Forms/FormsStorageDB.php

```diff
@@ -315,7 +315,12 @@ public function storeField(Field $field): Field
         $id = $field->getFieldId();
         $config = $field->getConfig();
 
-        $default_value = $config->getDefaultValue();
+        $default_value = (array) $config->getDefaultValue();
+        $options = $config->getOptions();
+        if (is_array($options) && !is_null($default_value)) {
+            $diff = array_diff($default_value, $options);
+            $default_value = array_diff($default_value, $diff);
+        }
```

Comment:
Das verstehe ich nicht. Ist das, um sicherzustellen, dass default_value in options enthalten ist?